Dockerfile

```diff
@@ -1,7 +1,13 @@
-# NAME: dclong/rustpython
```

Comment:
Would you keep the above comment line? 

I have an automated Docker image build tool which parses Dockerfiles to build dependencies graphs and then build Docker images in the right order. The above comment line acts as a metadata line defining the target Docker image. For a branch named `br`, the Docker image `dclong/rustpython:br` will be built.